test/e2e/run-all.js

```diff
@@ -212,12 +213,31 @@ async function main() {
 
   console.log('My test list:', myTestList);
 
+  let changedOrNewTests = '';
+  try {
+    changedOrNewTests = await fetchChangedE2eFiles();
+  } catch (error) {
+    console.error('Error fetching changed e2e files:', error);
+  }
```

Comment:
SUggestion: we can remove the catch here as it is handled in `fetchChangedE2eFiles`

Reply:
thank you for your suggestion. I've updated the `fetchChangedE2eFiles` so in the case something goes wrong it returns an empty string. This way we can remove this try/catch so we are always safe

Reply:
Thank you !